_posts/2021-02-00-the-life-science-community.md

```diff
@@ -0,0 +1,193 @@
+* Integrating Dask with the squidpy ImageContainer class is a good first step towards handling large image data within the availabe RAM constraints.
+
+[ilastik](https://www.ilastik.org/) does not currently use Dask at all. They are curious to see if Dask can make it easier to scale up from a single machine to a cluster.
+Users generally train an ilastik model interactively, and then want to apply it to many images. This second step is often when people want an easy way to scale up the computing resources available.
+
+[CellProfiler](https://cellprofiler.org/) is a pipeline tool for image processing. They have briefly experimented with Dask before.
+* Primarily, they want to parallelize existing functionality.
+* Most common pipelines fall into three major "user stories" where focussing effort would make the most impact:
+    1. Image processing
+    2. Object processing
+    3. Measurements
+
+## Opportunities we see
+
+Because large scientific software projects have many users, improvements here would be high value for the scientific community. This is a huge area of opportunity. We plan to collaborate with these developer communities as much as possible to drive this forward.
+
+Another area of opportunity is improving infrastructure for [high level graph visualizations](https://github.com/dask/dask/issues/7141). Power users and novices alike would benefit from better tools for identifying areas of inefficiencies in Dask computations.
+
+Finally, continuing to build support for Dask arrays with non-numpy chunks is also a high impact area of opportunity. In particular, support for sparse arrays, and support for arrays on the GPU were highlighted as very important to the life science community.
+
+## Strategic direction
+We're going to manage this project with three parallel streams:
+* [INFRASTRUCTURE](#Infrastructure) (60%)
+* [OUTREACH](#Outreach) (20%)
+* [APPLICATIONS](#Applications) (20%)
+
+Each stream will likely have one primary project at any time, with many more queued. Within each stream, proposed projects will be ranked according to: level of impact, time commitment required, and the availability of other developer resources.
+
+### Infrastructure
+Infrastructure projects are improvements to either:
+* Projects housed within the [Dask organisation](https://github.com/dask/), or
+* Other software projects involving Dask with large numbers of life science users
+
+We'll aim to spend around 60% of project effort on infrastructure.
+
+### Outreach
+Outreach activities include blogposts, talks, webinars, tutorials, and creating examples for documentation. We aim to spend around 20% of project effort on outreach.
```

Comment:
It occurs to me that readers of this post may have ideas for outreach activities.  For example maybe they run a student group or popular meetup.  Is there something that we can add here to let them engage productively with us?